src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -53,6 +84,61 @@ export function PharmacyCard(props: PharmacyProps) {
       styles={{ border: "none" }}
     />
   );
+  const generateRows = () => {
+    // TODO:
+    // We need to decide how we want to display availabilities if we have 1 vs many requirements on a given day.
+    // Below I've set it up to show a total along with a per-requirement breakdown.  If we want to keep this,
+    // we should do some styling cleanup for the user's benefit
+
+    const rows: (string | number)[][] = [];
+
+    Object.keys(props.vaccineAvailabilitiesNew).forEach((date) => {
+      rows.push([
+        format(new Date(date), "MMM d, y"),
+        props.vaccineAvailabilitiesNew[date].totalAvailable,
+      ]);
+
+      // TODO:
+      // Based off coversations with the backend team, there may be a need to differentiate vaccineAvailabilities
+      // by requirements on a given day.  If so, something like the below could work.  Styling total vs
+      // requirement-specific rows seems tricky with the default datatable, so we might need to find another
+      // option or build a custom table
+
+      // props.vaccineAvailabilitiesNew[date].requirements.forEach(
+      //   (requirement) => {
+      //     rows.push([requirement.description, requirement.numberAvailable]);
+      //   },
+      // );
+    });
+
+    return rows;
+  };
+  const dataTableMarkup = shouldShowSlots ? (
```

Comment:
Are we going to break down amount by eligibility requirement? Or is that coming in a future ticket?